copy.c

```diff
@@ -348,7 +348,8 @@ int mutt_copy_hdr(FILE *in, FILE *out, LOFF_T off_start, LOFF_T off_end,
       if (flags & (CH_DECODE | CH_PREFIX))
       {
         if (mutt_write_one_header(out, 0, headers[x], (flags & CH_PREFIX) ? prefix : 0,
```

Comment:
This is unrelated to the naming, but this `if` statement should be modified to be more read able. Moving `(flags & CH_PREFIX) ? prefix : 0` and `mutt_window_wrap_cols(...)` out will make it significantly easier to understand.

Reply:
Yes, definitely.  There's a lot of code like this.
I guess the condition started more simply and evolved.

I've made a note.  I usually keep a `trivial` branch on the neomutt repo to collect these important little fixes.  Feel free to create one and dump stuff in it.

Reply:
Duly noted.